node/malus/Cargo.toml

```diff
@@ -24,7 +24,6 @@ polkadot-cli = { path = "../../cli", default-features = false, features = [
 polkadot-node-subsystem = { path = "../subsystem" }
 polkadot-node-subsystem-util = { path = "../subsystem-util" }
 polkadot-node-core-candidate-validation = { path = "../core/candidate-validation" }
-# parity-util-mem = { version = "0.10.0", default-features = false, features = ["jemalloc-global"] }
```

Comment:
I believe it was added here on purpose to enable jemalloc and have similar execution environment to the polkadot binary. cc @drahnr 
If that's the case, we could add a comment here stating the purpose.

Reply:
I think the change is sane, but we should include a note where `parity-util-mem` is included, that `jemalloc-ctl` crate is used on `*nix` platforms, so changes have to be applied with care.